### ISSUE TITLE
export default from '...'

### ISSUE THREAD

**Issue by benmosher** [benmosher/eslint-plugin-import#42]
verify this shows up as a default export in an importing module, i.e.

```js
// mod.js
export default function () {}

// bar.js
export default from './mod'

// foo.js
import default from './bar' // <= should not report for import/default
```

**Comment by benmosher** [comment]
Need to add tests for 
```js
// mod.js
export x from './x'
export y from './y'

// importer.js
import { x, y } from './mod'
```

Something in the last few commits broke this case without breaking any tests.
